translations/game_modes.tex/pl.po

```diff
@@ -86,7 +86,7 @@ msgid ""
 "\n"
 msgstr ""
 "\\subsection*{Starcie}\n"
-"Tryb rywalizacji dla 2-3 graczy.\n"
+"Tryb rywalizacji dla 2-3 Bohatery.\n"
```

Comment:
chyba za dużo replaceów? spójrz też poniżej

Reply:
a to już nie mój plik... ale faktycznie chyba poszły zmiany z automatu...